src/app/index.js

```diff
@@ -1,117 +1,107 @@
-const gui = require('nw.gui');
 const gkm = require('gkm');
-const pjson = require('../package.json');
 const AudioStreamMeter = require('audio-stream-meter');
-const character = document.getElementById("character");
-var exportable = require("./twitchAuth.js");
 
-//Tesing twitch auth file
-exportable.doSomething();
+const exportable = require('../app/twitchAuth.js');
+
+const character = document.getElementById('character');
+
+// Tesing twitch auth file
+exportable.doSomething(window);
```

Comment:
When you `require` something, that code lives in the Node context. so if you want that code to be able to access or manipulate the DOM, you need to pass it the `window` object.